modular_skyrat/modules/opposing_force/code/opposing_force_datum.dm

```diff
@@ -922,35 +922,37 @@
 						if(!opfor_objective)
 							continue
 
-						set_objective_title(importer, opfor_objective, opfor_data["objectives"][iter_num]["title"])
-						set_objective_description(importer, opfor_objective, opfor_data["objectives"][iter_num]["description"])
-						set_objective_justification(importer, opfor_objective, opfor_data["objectives"][iter_num]["justification"])
-						set_objective_intensity(importer, opfor_objective, opfor_data["objectives"][iter_num]["intensity"])
+						var/list/iter_obj = opfor_data["objectives"][iter_num]
+
+						set_objective_title(importer, opfor_objective, iter_obj["title"])
+						set_objective_description(importer, opfor_objective, iter_obj["description"])
+						set_objective_justification(importer, opfor_objective, iter_obj["justification"])
+						set_objective_intensity(importer, opfor_objective, iter_obj["intensity"])
 
 				if("backstory")
 					set_backstory(importer, opfor_data["backstory"])
 
 				if("selected_equipment")
 					for(var/iter_num in opfor_data["selected_equipment"])
 						// If there isn't category data / a given equipment type, OR if either of those don't fit within certain perameters, it continues
+						var/list/iter_eqpmt = opfor_data["selected_equipment"][iter_num]
```

Comment:
Abbreviations like these just make it harder to read. You can remove the "iter" prefix, it isn't really necessary.

Reply:
done